docs/adr/adr-0026-introducing-prettier.md

```diff
@@ -0,0 +1,161 @@
+- Biome has the same philosophy as Prettier for the formatting but Prettier is more known and used for now.
+
+### dprint
+
+From [dprint's website](https://dprint.dev/):
+
+> A pluggable and configurable code formatting platform written in Rust.
+
+#### Pros
+
+- Lot of options.
+- You can develop plugins.
+- If you want to develop plugins, there's a rust SDK.
+
+#### Cons
+
+- Even though it's possible to write plugins in another language, resources about this are sparse. It means you better be proficient with Rust.
+
+### Prettier
+
+[Prettier](https://prettier.io) is an opiniated code formatter.
+
+#### Pros
+
+- Can format both JS, CSS, template literals in one take.
+
+#### Cons
+
+- There are not a lot of options which is justified by the `opiniated formatter` logic.
+
+Prettier is the option we chose to use, you can find more information about the why within the dedicated section below.
+
+### js-beautify
+
+#### Pros
+
+- Has integrations with IDEs
+
+#### Cons
+
+- Unsure about the future of the maintenance ([See github README](https://github.com/beautifier/js-beautify?tab=readme-ov-file#contributors-needed))
+- Can over-format
+
+### Others
+
+- While we knew all these tools existed we wanted to know what Lit users were using
+  - Knowing that we asked the Lit discord's what they were using
+    - Someone told us that they were using the new `Stylistic` package for ESLint that keeps formatting rules
+    - They were also using a Prettier plugin for CSS
+    - While it seemed a good idea at first, you'll find why it wasn't adapted to our case below.
+
+## Solution
+
+### Why did we choose Prettier?
+
+We decided to go for Prettier for various reasons:
+
+- It is widely used and maintained.
+- Has integration with IDEs.
+- It enforces code style. Other than our internal conventions we didn't have a proper formatter enforcing.
+
+Note: As we haven't updated ESLint yet, see details on the dedicated section below, we've added the Prettier ESLint plugin so that ESLint formatting rules doesn't conflict with Prettier.
+
+### Why didn't we use ESLint Stylistic?
+
+At some point we considered going for the `stylistic` package for ESLint to keep our usage.
+While it would have been great, this wouldn't solve the lack of formatter for Stylelint.
+
+Considering this, we tried to use the `stylelint-prettier` plugin to format only our CSS.
+However, we couldn't do that because:
+`postcss-lit` uses babel and if you use that parser `stylelint-prettier` will just skip your file.
+So we tried to force Prettier to use the `css` parser instead for Stylelint but that resulted in some errors.
+
+Knowing that we couldn't use this plugin and thus not having a formatter for our Stylelint/CSS part we decided not to use Stylistic.
+Moreover, if we wanted to use both ESLint with Stylystic and prettier for the CSS it would have caused some conflicts.
```

Comment:
**🔨 issue:**

```suggestion
Moreover, if we wanted to use both ESLint with Stylistic and prettier for the CSS it would have caused some conflicts.
```